### PR TITLE
Ported FMI "feels like" temperature calculation.

### PR DESCRIPTION
**Pull request by jaenis**
Added temperature "feels like" calculation based on FMI's original C code.
Bumped version to 0.2.1

### CODE REVIEW

File: fmi_weather_client/parsers/forecast.py [hunk 1]
```diff
@@ -138,10 +138,55 @@ def to_value(vals: Dict[str, float], variable_name: str, unit: str) -> Value:
...
+
+    # Perform radiation correction only when radiation is available
+    if rad is not None:
+        absorption = 0.07
+        feels += 0.7 * absorption * rad / (w + 10) - 0.25
+
+    return feels
+
+
+def _summer_simmer(t: float, rh: float):
```
Comment (saaste) [inline fmi_weather_client/parsers/forecast.py:178]: Same here. I think using `temp` and `humidity` makes the code a bit more readable :relaxed:

**Review by saaste — CHANGES_REQUESTED** [review]
Thank you for the contribution and sorry for the delay! I was on vacation so took a while to check GitHub :relaxed: 

It looks like the older version of `pylint` has a bug that makes it hate short variable names. And to be honest, I'm not a big fan of them either, except in loops :smile: 

The newer pylint fixes the issue, but it does not support Python 3.7. I could drop the support of 3.7 a bit later since it is already in `end-of-life` state, but for the time being I'd appreciate if you could change the code so that it makes the older pylint happy :relaxed:

**Comment by saaste** [comment]
One more request: could you please update README.md? "Feels like" is such a useful field it could be mentioned under "FMI provides the following commonly used information" :relaxed:

File: fmi_weather_client/parsers/forecast.py [hunk 2]
```diff
@@ -138,10 +138,55 @@ def to_value(vals: Dict[str, float], variable_name: str, unit: str) -> Value:
...
         )
 
 
+def _feels_like(vals: Dict[str, float]) -> float:
+    # Feels like temperature, ported from:
+    # https://github.com/fmidev/smartmet-library-newbase/blob/master/newbase/NFmiMetMath.cpp#L535
+    # For more documentation see:
+    # https://tietopyynto.fi/tietopyynto/ilmatieteen-laitoksen-kayttama-tuntuu-kuin-laskentakaava/
+    # https://tietopyynto.fi/files/foi/2940/feels_like-1.pdf
+    t = vals.get("Temperature", None)
```
[the remark this answers is not in the document]
Comment (jaenis) [reply fmi_weather_client/parsers/forecast.py]: When I ported the original code to python I did stick with short names - and did not get errors from pylint during tests, so did stick with those :) 
Now replaced with longer names

File: fmi_weather_client/parsers/forecast.py (hunk 1 above)
Comment (jaenis) [reply fmi_weather_client/parsers/forecast.py]: Done

**Comment by jaenis** [comment]
Did updates. Should they be squashed into one commit - or kept as separate? Really up to you 😃

**Comment by saaste** [comment]
Awesome! Looks good! I'll merge this and release a new version after my workday ☺️ 

Thanks again for the contribution! 🙏

**Comment by jaenis** [comment]
Thanks!

**Pull request merged**